testing/functional/tests/scheduler/resizeAppointments/basic.ts

```diff
@@ -58,4 +62,8 @@ test(`Resize in the "month" view`, async t => {
         .expect(resizableAppointment.date.startTime).eql(`10:00 AM`)
         .expect(resizableAppointment.date.endTime).eql(`11:00 AM`);
 
-}).before(() => createScheduler('month', dataSource));
+}).before(() => createScheduler({
+    views: ['month'],
```

Comment:
What do you think about removing this 'views: [view]' argument?

Reply:
I think that tests must satisfy a "Single Responsibility Principle", which says that any function must serve only one purpose. In that case, test will be "broken" if someone will break a logic referring to `views`. So, the test will `false-trigger` on the problem which has no common with `Resizing`

Reply:
I understood.